src/Facilitator/index.js

```diff
@@ -351,6 +356,7 @@ class Facilitator {
     nonce,
     hashLock,
     unlockSecret,
+    blockNumber,
```

Comment:
Two points on adding a parameter 
1. Should there be a validation of undefined for block number like other paramters
2. Should this parameter be added at the last so that it can have default value `latest` if not passed?

Reply:
1. 💯 Done!
2. That can create confusion😕 🤔  . With the expectation of exact block number in input, it's clear that progress stake needs to be done at exact block height.

Reply:
On 2nd, it was with regards to backward compatibility. If you add a parameter at last and give it a default value it will be backward compatible. 
I don't have a strong opinion on this.